File: server-product-management/src/main/java/com/sha/serverproductmanagement/model/DataAnnotation.java
package com.sha.serverproductmanagement.model;

import lombok.Data;

import javax.persistence.*;

@Data
@Entity
@Table(name="dataAnnotation")
public class DataAnnotation {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name="category_id")
    private String category_id;

    @Column(name="is_crowd")
    private String is_crowd;

    @Column(name="segmentation",columnDefinition="TEXT")
    private String segmentation;

    @Column(name="image_id")
    private Long image_id;

    @Column(name="area")
    private double area;



}

File: client-product-management/src/app/model/dataAnnotation.ts
export class DataAnnotation {
  id:number;
  // tslint:disable-next-line:variable-name
  category_id: number;
  // tslint:disable-next-line:variable-name
  is_crowd:boolean;
  segmentation:string;
  // tslint:disable-next-line:variable-name
  image_id:number;
  area:string;
  constructor(id,imageId,segmentation) {
    this.id = id;
    this.image_id = imageId;
    this.segmentation = segmentation.toString();
  }
}
